app/etl/item/metadata/parsers/metadata.rb

```diff
@@ -7,6 +7,8 @@ def self.parse(formatted_response)
       'title',
       'document_type',
       'content_purpose_document_supertype',
+      'content_purpose_supergroup',
```

Comment:
Could you please include these two fields in the CSV export from the Sandbox?

Reply:
Done